UnityProject/Assets/Scripts/Systems/Lobby/CharacterManager.cs

```diff
@@ -272,8 +272,18 @@ public async Task LoadOnlineCharacters()
 					{
 						Loggy.Error(
 							$"Failed to load characters online. because: {accountResponse.Exception!.Message}");
+						UIManager.InfoWindow.Show("Failed to load characters online" +
+						                          $"{accountResponse.Exception!.Message}",
+							false, "Error");
 					});
-					throw accountResponse.Exception;
+					if (accountResponse.Exception != null)
```

Comment:
there already is an error logged above. No need to throw an exception that won't be handled

Reply:
Rider shows an error if its left without a null check

Reply:
why not handle log a big error but generate a new character?

Reply:
because it's not always about generating a new character, it could be a network error